crt_portal/cts_forms/forms.py

```diff
@@ -856,6 +857,35 @@ def clean(self):
             return cleaned_data
 
 
+class ProfileForm(ModelForm):
+    intake_filters = MultipleChoiceField(
+        required=False,
+        choices=SECTION_CHOICES_WITHOUT_LABELS,
+        widget=UsaCheckboxSelectMultiple(attrs={
+            'name': 'intake_filters'
+        })
+    )
+
+    class Meta:
+        model = Profile
+        fields = [
+            'intake_filters'
+        ]
+
+        labels = {
+            'intake_filters': 'View sections'
+        }
+
+    def clean_intake_filters(self):
+        """
+        Clean intake_filters by removing list markup
+        """
+        if 'intake_filters' in self.changed_data:
```

Comment:
`changed_data` will always be true here, since we don't pass in any `initial` data. I wonder if this was a typo and if this should be `cleaned_data` instead. Either way I think this function will continue to work, but its an oddity.